static/templates/widgets/todo_widget_tasks.hbs

```diff
@@ -1,7 +1,7 @@
 <br />
 {{#each pending_tasks}}
     <li>
-        <label class="checkbox">
+        <label id="checkbox_todo" class="checkbox">
```

Comment:
It isn't valid HTML to use IDs for elements that appear more than once in the DOM.  A CSS class is appropriate, but as you see we already have one -- so you should be able to just scope your CSS properly as e.g. `.todo_widget .checkbox` without adding this.

Reply:
So should I create a separate CSS class `.checkbox` under .todo_widget ?
I was thinking of another way, changing `#checkbox_todo` to `.checkbox_todo` and then using it in the following way: 
```
<label class="checkbox_todo checkbox">
```
Considering the CSS hierarchy this also provides the same result through overriding the existing `.checkbox` without the use of IDs

Reply:
That sound good -- and then we should remove the change adding this ID.